.vimrc

```diff
@@ -2,3 +2,4 @@ set tabstop=4
 set softtabstop=4
 set shiftwidth=4
 set colorcolumn=121
+autocmd BufWritePre * %s/\s\+$//e
```

Comment:
shouldn't be in repo but shouldn't affect either

Reply:
For the #TeamVim